src/Corax/IndexSearcher.cs

```diff
@@ -537,7 +548,21 @@ public MultiTermMatch StartWithQuery<TScoreFunction>(string field, string startW
 
             return MultiTermMatch.Create(
                 MultiTermBoostingMatch<StartWithTermProvider>.Create(
-                    this, new StartWithTermProvider(this, _transaction.Allocator, terms, field, 0, startWith), scoreFunction));
+                    this, new StartWithTermProvider(this, _transaction.Allocator, terms, field, startWith), scoreFunction));
+        }
+
+        public MultiTermMatch ContainsQuery<TScoreFunction>(string field, string containsTerm, TScoreFunction scoreFunction)
```

Comment:
This is a prefix search, why is it called contains?

Reply:
Because it is actually `Contains` . Say you have the terms `runner`, `running` and then do a contains for `nn` then you will get both.